graphql/src/schema/api.rs

```diff
@@ -263,6 +263,11 @@ fn add_order_by_type(
 
     match ast::get_named_type(schema, &type_name) {
         None => {
+            // Don't generate empty enums.
+            if fields.is_empty() {
+                return Ok(());
+            }
+
```

Comment:
If I remember the API generation code correctly, there's another place where the `_orderBy` type is referenced (where the top-level query fields are generated). That will now refer to a non-existent `_orderBy` type, which will probably break schema introspection as well.

Even if I don't like special-casing, we may have to add a static/hard-coded list of types for which to do things a little differently.

Reply:
I've changed the approach and went a bit further to completely remove the `_Schema_` type from the generated schema after all associated information from attached directives has been extracted. I think that's the saner approach anyways considering that the `_Schema_` type seems to only be used for metadata / configuration purposes.

The introspection works now also when there are full-text search fields.